src/views/ContributorsView.vue

```diff
@@ -0,0 +1,151 @@
+  </p>
+
+  <Role :rank="RANK.ModTier1" />
+  <p>Not Sentry</p>
+
+  <hr />
+
+  <SubHeading
+    title="Patreon Supporters"
+    imageSource="/images/Clan_icons/Bank.png"
+  />
+
+  <Role :rank="RANK.PatreonDragonBotHunters" />
+  <p v-if="!isArrayEmpty(configStore.DragonBotHunter)">
+    {{ configStore.DragonBotHunter.join(", ") }}
+  </p>
+  <p v-if="isArrayEmpty(configStore.DragonBotHunter)">
+    <ExternalLink to="https://www.patreon.com/bot_detector">
+      [This slot could be yours!]
+    </ExternalLink>
+  </p>
+
+  <Role :rank="RANK.PatreonRuneBotHunters" />
+  <p v-if="!isArrayEmpty(configStore.RuneBotHunter)">
+    {{ configStore.RuneBotHunter.join(", ") }}
+  </p>
+  <p v-if="isArrayEmpty(configStore.RuneBotHunter)">
+    <ExternalLink to="https://www.patreon.com/bot_detector">
+      [This slot could be yours!]
+    </ExternalLink>
+  </p>
+
+  <Role :rank="RANK.PatreonAdamantBotHunters" />
+  <p v-if="!isArrayEmpty(configStore.AdamantBotHunter)">
+    {{ configStore.AdamantBotHunter.join(", ") }}
+  </p>
+  <p v-if="isArrayEmpty(configStore.AdamantBotHunter)">
+    <ExternalLink to="https://www.patreon.com/bot_detector">
+      [This slot could be yours!]
+    </ExternalLink>
+  </p>
+
+  <Role :rank="RANK.PatreonMithrilBotHunters" />
+  <p v-if="!isArrayEmpty(configStore.MithrilBotHunter)">
+    {{ configStore.MithrilBotHunter.join(", ") }}
+  </p>
+  <p v-if="isArrayEmpty(configStore.MithrilBotHunter)">
+    <ExternalLink to="https://www.patreon.com/bot_detector">
+      [This slot could be yours!]
+    </ExternalLink>
+  </p>
+
+  <Role :rank="RANK.PatreonSteelBotHunters" />
+  <p v-if="!isArrayEmpty(configStore.SteelBotHunter)">
+    {{ configStore.SteelBotHunter.join(", ") }}
+  </p>
+  <p v-if="isArrayEmpty(configStore.SteelBotHunter)">
+    <ExternalLink to="https://www.patreon.com/bot_detector">
+      [This slot could be yours!]
+    </ExternalLink>
+  </p>
+
+  <Role :rank="RANK.PatreonIronBotHunters" />
+  <p v-if="!isArrayEmpty(configStore.IronBotHunter)">
+    {{ configStore.IronBotHunter.join(", ") }}
+  </p>
+  <p v-if="isArrayEmpty(configStore.IronBotHunter)">
+    <ExternalLink to="https://www.patreon.com/bot_detector">
+      [This slot could be yours!]
+    </ExternalLink>
+  </p>
+
+  <Role :rank="RANK.PatreonBronzeBotHunters" />
+  <p v-if="!isArrayEmpty(configStore.BronzeBotHunter)">
+    {{ configStore.BronzeBotHunter.join(", ") }}
+  </p>
+  <p v-if="isArrayEmpty(configStore.BronzeBotHunter)">
+    <ExternalLink to="https://www.patreon.com/bot_detector">
+      [This slot could be yours!]
+    </ExternalLink>
+  </p>
+
+  <Role :rank="RANK.PatreonStarterBotHunters" />
+  <p v-if="!isArrayEmpty(configStore.starterBotHunter)">
+    {{ configStore.starterBotHunter.join(", ") }}
+  </p>
+  <p v-if="isArrayEmpty(configStore.starterBotHunter)">
+    <ExternalLink to="https://www.patreon.com/bot_detector">
+      [This slot could be yours!]
+    </ExternalLink>
+  </p>
+</template>
```

Comment:
I saved this comment for this template in particular, including whitespace in any flavour of HTML component is a performance hit, especially at compile time. All that will happen is these spaces get removed and minified anyway. So this formatting is not recommended from a user quality of life standpoint.

Reply:
the gaps between tags etc, they all enter the parser and while only being a small performance hit, a lot of this can have a a larger impact over the course of the project etc, would also help with consistency of formatting etc

Reply:
I do agree to a certain level but long building time shouldnt be too much of a problem. I don't think the project will get any bigger right now. I also rather have more readability over compiling performance. About formatting a portion of the formatting is also done by the project automatically which I don't always have control over it maybe it can be done in the config somewhere but the intention of the PR is mostly removing the vulnerabilities and refactoring the code to have a start of a decent structure.